{
    "發生未知錯誤，請稍後再試": "unknown_error",
    "測試鑰匙": "test_key",
    "測試成功":"test_success",
    "未知請求將導向外部網址":"OIIA OIIA",
    "[全域錯誤]":"[Global Error]",
    "未知錯誤":"Unknown Error",
    "❌ 缺少環境變數 DB_URL":"❌ Missing environment variable: DB_URL",
    "✅ 資料庫連線成功":"✅ 資料庫連線成功",
    "🚀 伺服器啟動：port ":"🚀 server_started ",
    "❌ 資料庫連線失敗：":"❌ Failed to connect to the database:"
}